### PR TITLE
Update to publish packages to github maven repo.

### PR DESCRIPTION
**Pull request by corneil**

### CODE REVIEW

**Comment by bclozel** [comment]
Thanks for the PR, but we usually don't publish to such repositories.

Have you seen #71 and the solutions listed there?

**Comment by corneil** [comment]
Spring Cloud Data Flow documentation is still using it, and cannot access repo.spring.io/release since it was closed. 
Publishing it using this mechansim will make it available to anyone still using this.

**Comment by corneil** [comment]
Spring Cloud Data Flow documentation is still using it, and cannot access repo.spring.io/release since it was closed. 
Publishing it using this mechansim will make it available to anyone still using this.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'liberica'
+      - name: Build and Publish
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_ACTOR: ${{ secrets.OSSRH_TOKEN }}
```
Comment (bclozel) [inline .github/workflows/ci.yml]: is this a typo?

Comment (corneil) [reply .github/workflows/ci.yml]: Yes

**Pull request closed without merging**